framework/src/abi/abi.ts

```diff
@@ -172,6 +172,7 @@ export interface VerifyTransactionRequest {
 
 export interface VerifyTransactionResponse {
 	result: number;
+	errorMessage?: string;
```

Comment:
```suggestion
	errorMessage: string;
```
There is no optional field in ABI